therapy/etl/base.py

```diff
@@ -3,8 +3,10 @@
 from therapy.schemas import NamespacePrefix
 from therapy.database import Database
 
+# prefixes for translating ID namespaces
 IDENTIFIER_PREFIXES = {
     'casRegistry': NamespacePrefix.CASREGISTRY.value,
+    'ChemIDplus': NamespacePrefix.CHEMIDPLUS.value,  # needed for wikidata
     'pubchemCompound': NamespacePrefix.PUBCHEMCOMPOUND.value,
     'pubchemSubstance': NamespacePrefix.PUBCHEMSUBSTANCE.value,
     'chembl': NamespacePrefix.CHEMBL.value,
```

Comment:
It seems like DrugBank actually stores the values as 'ChemIDplus' by default and there's an extra method converting them to "casRegistry" -- it might be simpler to just remove that logic and delete line 8 here (that way we don't have to hang onto prefixes with different sourcenames than our internal values) but I feel like that maybe should be its own PR